BaseClasses.py

```diff
@@ -455,6 +455,22 @@ def has_Pearl(self, player):
     def has_fire_source(self, player):
         return self.has('Fire Rod', player) or self.has('Lamp', player)
 
+    def can_flute(self, player):
+        lw = self.world.get_region('Light World', player)
+        return self.has('Ocarina', player) and lw.can_reach and self.is_not_bunny(lw, player)
```

Comment:
lw.can_reach is a function, so you should invoke it, otherwise it will always be true. (I've done the same thing myself on a few occasions, to much head scratching when i some across the unbeatable seed.)

Reply:
Fixed